app/services/piece-access-level-service.js

```diff
@@ -80,7 +80,8 @@ export default class PieceAccessLevelService extends Service {
     });
 
     await Promise.all(pieces.toArray().map((piece) => {
-      if (piece.accessLevel.get('uri') !== ministerraad.uri) {
+      const accessLevel = await piece.accessLevel;
```

Comment:
Mag je hier `await` gebruiken zonder de lambda-functie ook `async` te maken?

Reply:
gaat inderdaad niet, ben vanochtend te snel geweest met de aanpassing. thanks for the catch!